packages/utils/src/commonwell/doc-query-shared.ts

```diff
@@ -58,10 +56,15 @@ export async function queryDocsForPatient({
   const metriportAPI = new MetriportMedicalApi(apiKey, {
     baseAddress: apiUrl,
   });
+
+  const metadata = {
+    metadata: {
+      disableWHFlag: "true",
```

Comment:
Why are we always disabling it? There was a parameter for the exact reason to pass this from the code calling this function. The TODO was talking about the directive, not the parameter, as we also talked about in the huddle.

The code is not compiling, try to compile `package/utils`. I don't know why it was not spotted by CI, looking into it.